client/boot/project/wordpress-com.js

```diff
@@ -85,7 +87,11 @@ export function setupMiddlewares( currentUser, reduxStore ) {
 	// Render Layout only for non-isomorphic sections.
 	// Isomorphic sections will take care of rendering their Layout last themselves.
 	if ( ! document.getElementById( 'primary' ) ) {
-		renderLayout( reduxStore );
+		page( '*', function( context, next ) {
```

Comment:
You need to perform rendering on the server and then repeat on the client when SSR is enabled. Technically React discovers that DOM hasn't changed and skips repaint and reflow phase. I think we shouldn't introduce this change.